src/main/kotlin/org/partiql/scribe/targets/redshift/RedshiftDialect.kt

```diff
@@ -77,6 +84,77 @@ public open class RedshiftDialect : SqlDialect() {
 
     override fun visitTypeString(node: Type.String, tail: SqlBlock): SqlBlock = tail concat "VARCHAR"
 
+    // Essentially the same as `SqlDialect`. For SQL time and timestamp literals, Redshift (like Postgresql) requires
+    // specifying if the time/timestamp has a timezone in the type name. Otherwise, the offset will be ignored.
+    // Postgresql reference -- https://www.postgresql.org/docs/16/datatype-datetime.html#DATATYPE-DATETIME-INPUT-TIME-STAMPS
```

Comment:
Comment summarizes the changes for Redshift time and timestamp w/ offset translation. Date remains the same.